app/models/katello/concerns/setting_extensions.rb

```diff
@@ -11,7 +11,7 @@ module SettingExtensions
       end
 
       def recalculate_errata_status
-        ForemanTasks.async_task(Actions::Katello::Host::RecalculateErrataStatus) if value_changed? && name == 'errata_status_installable'
+        ForemanTasks.async_task(Actions::Katello::Host::RecalculateErrataStatus) if value_previously_changed? || value_changed? && name == 'errata_status_installable'
```

Comment:
Parenthesis must be added for this still to work properly:

```ruby
(value_previously_changed? || value_changed?) && name == 'errata_status_installable'
```

Reply:
yes, fixed that haven't committed it yet